.gitlab-ci.yml

```diff
@@ -20,13 +20,21 @@ default:
   rules:
     - if: $BUILD
       when: never
-    - if: "$CI_MERGE_REQUEST_ID =~ /^./ || $CI_COMMIT_BRANCH == 'master'"
+    - if: $CI_COMMIT_REF_NAME == "master"
+    - if: $CI_COMMIT_REF_NAME =~ /^[0-9]+$/                         # PRs
+
+.on-tags: &on-tags
+  rules:
+    - if: $BUILD
+      when: never
```

Comment:
The `- if: $BUILD` should be deleted. `$BUILD` is supposed to always work for protected branches since I only set it on manual pipeline triggers.

Reply:
👌